doc/api/zlib.md

```diff
@@ -385,6 +385,12 @@ added: v0.5.8
 Not exported by the `zlib` module. It is documented here because it is the base
 class of the compressor/decompressor classes.
 
+### zlib.bytesRead
+
+* {number}
+
+The `zlib.bytesRead` property specifies the number of bytes read by the engine.
```

Comment:
`bytes in -> zlib engine -> bytes out`: this is the bytes read by the engine from the encrypted stream (bytes out), or the other side (bytes in)? I suspect its bytes in, but it should be overwhelmingly clear from the docs, please, perhaps "bytes read by the engine before the bytes are processes (compressed or decompressed, as appropriate for the derived class)" <--- or change the language if its bytes read post processing.